### ISSUE TITLE
Issues using as a remote-theme hosted on github pages

### ISSUE THREAD

**Issue by faaip** [bk2dcradle/researcher#7]
Hey @ankitsultana , thank you for a nice theme. Very glad to use it!

I'm trying to migrate to github pages and wish to use your repo as a remote-theme. When trying to use it as a remote theme I get:

`Invalid theme folder: _sass
      Remote Theme: Using theme ankitsultana/researcher
jekyll 3.8.3 | Error:  404 - Not Found`

Is the theme intended to be used remotely and if yes, how so? If not I'd gladly give a hand preparing it to be used in such a manner.

Sincerely,
Frederik

**Comment by ankitsultana** [comment]
Hi!

Actually, I have been thinking of doing this for a while now. I will get it done by this Sunday.

**Comment by ankitsultana** [comment]
Hi! I had actually been thinking about this for a while. For the moment, I have added the remote theme feature in the gem branch of this project. I have tested it out and it seems to be working. I will soon merge that branch into gh-pages and update the README with the updated instructions.

tl;dr : What you are intending to do will start working in the next 24 hours.

**Comment by faaip** [comment]
Perfect, looking forward to the merge! Thank you for your great work!

**Comment by ankitsultana** [comment]
So I couldn't find time to get to it today. Will try and close this by the weekend. For the moment, the gem branch works perfectly fine. As in, you can, for now, create a repo and have just the file `_config.yml` in its `gh-pages` with `remote_theme: ankitsultana/researcher@gem`, and it will work.

**Comment by ankitsultana** [comment]
I had thought that we can build remote themes locally for testing purposes. It seems I was mistaken. If the remote themes are intended to be built only on remote `gh-pages` branch, then this issue is fixed.

Bottom line: If you want to use this theme as a remote theme, setting `remote_theme: ankitsultana/researcher@gem` in `_config.yml` of your repo's `gh-pages` branch will work.

**Comment by faaip** [comment]
Dear @ankitsultana . Thank you for your help, I am however struggling with getting it to work, which is maybe due to trying to run it as user-based github page. Could you by any chance share a repository, where you use your theme as a remote-theme. Just the initial config and such. Sorry for the hassle!

**Comment by ankitsultana** [comment]
Sure. Check out my [scripts](https://github.com/ankitsultana/scripts/tree/gh-pages) repo. [Here](http://ankitsultana.com/scripts/) is the web link. I have just added _config.yml with `remote_theme: ankitsultana/researcher@gem`.
